server/package.json

```diff
@@ -37,7 +37,7 @@
     "eslint": "eslint --max-warnings 0 \"./src/**/*.ts\" \"./test/**/*.ts\"",
     "prettier": "prettier \"*.json\" \"src/**/*.{ts,js,md,json,yml}\" \"test/**/*.{ts,js,md,json,yml}\"",
     "clean": "rimraf out .nyc_output coverage *.tsbuildinfo",
-    "test:codecov": "cross-env TS_NODE_FILES=true nyc --reporter=lcov mocha && codecov"
```

Comment:
This script is now just about running tests with coverage and the appropriate reporter for codecov.

A follow on github action is responsible for the coverage upload.